tests/test_closing.py

```diff
@@ -241,7 +240,7 @@ def test_closing_different_fees(node_factory, bitcoind, executor):
     # so increase the timeout so that this test will pass
     # when valgrind is enabled.
     # (close timeout defaults to 30 as of this writing)
```

Comment:
Maybe change the comment as well?

Reply:
Actually, changed the arg to 0, so it's infinite and no unilateral.

Reply:
Latest commit-set seems to have skipped this use of `close` entirely?

Reply:
No, it eliminates it?

```diff
-    # Now close all channels
-    # All closes occur in parallel, and on Travis,
-    # ALL those lightningd are running on a single core,
-    # so increase the timeout so that this test will pass
-    # when valgrind is enabled.
-    # (close timeout defaults to 30 as of this writing)
-    closes = [executor.submit(l1.rpc.close, p.channel, False, 90) for p in peers]
+    # Now close all channels (not unilaterally!)
+    closes = [executor.submit(l1.rpc.close, p.channel, 0) for p in peers]
```

Reply:
Hmm, must have been some weirdness in github then.  Okay.